README.md

```diff
@@ -1,11 +1,160 @@
+[![Hex pm](https://img.shields.io/hexpm/v/kaur.svg)](https://hex.pm/packages/kaur)
+[![Build
+Status](https://travis-ci.org/fewlinesco/kaur.svg?branch=master)](https://travis-ci.org/fewlinesco/kaur)
+[![Inline docs](http://inch-ci.org/github/fewlinesco/kaur.svg)](http://inch-ci.org/github/fewlinesco/kaur)
+
 # Kaur
 
-A bunch of helper functions we use in different projects.
+A bunch of helper functions which ease the development of your applications.
 
 ## Installation
 
 ```elixir
 def deps do
-  [{:kaur, github: "fewlines/kaur"}]
+  [{:kaur, "~> 0.1.0"}]
```

Comment:
Isn't the first version of `kaur` going to be `1.0.0`?

Reply:
as you wish.